runestone/activecode/activecode.py

```diff
@@ -229,6 +232,60 @@ def run(self):
         if 'gradebutton' not in self.options:
             self.options['gradebutton'] = ''
 
+        if self.content:
+            if '====' in self.content:
+                idx = self.content.index('====')
+                source = "\n".join(self.content[:idx])
+                suffix = "\n".join(self.content[idx+1:])
+            else:
+                source = "\n".join(self.content)
+                suffix = "\n"
+        else:
+            source = '\n'
+            suffix = '\n'
+        try:
+            engine = create_engine(env.config['dburl'])
+            meta = MetaData()
+            course_name = env.config['course_id']
```

Comment:
Yep, I double checked my comment, and the conf.py file is not saved/customized for each custom built course.  So, adding both the dburl to pavement.py and using the course_id that is already in pavement.py is a much better way to go from my perspective.